Libraries/libutil/Headers/libutil/ProcessContext.h

```diff
@@ -85,7 +85,7 @@ class ProcessContext {
     /*
      * Get the system instance.
```

Comment:
Maybe add a note that the caller should strive to use a context passed from above whenever possible.